package.json

```diff
@@ -51,6 +51,7 @@
     "bl": "^1.1.2",
     "chai": "^3.5.0",
     "css-loader": "^0.23.1",
+    "engine.io-client": "^1.6.9",
```

Comment:
Didn't this only just get removed here - https://github.com/numo-labs/isearch-ui/commit/0a93c5a1204938c9b1b38877ac3f25a5afee0e21? How come it's come back?